python/cugraph/structure/replication.pyx

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Should this be under `dask`?

Reply:
Moved it to `python/cugraph/dask/structure/replication.pyx`